.github/workflows/react_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: react-tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    paths:
+    - client/
+    - **.js
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: yarn
+    - name: Yarn Build
+      run: yarn build
+    - name: Yarn Test
+      run: yarn test --watchAll=false
```

Comment:
Filed https://github.com/joshprzybyszewski/cribbage/issues/64 because we don't have any react tests for this new workflow to run

Reply:
:oof: